### PR TITLE
Move poke module to src/

### LINKED ISSUE #8498: Redirect after poking/pinging someone doesn't take place
### Expected behavior
The website redirects to previous page.

### Actual behavior
The website remains after the `GET` (shouldn't it be `POST`?) request finishes. It may allow to press F5 (reload) very often.

### Steps to reproduce the problem
Ping/poke a Friendica account.

### Friendica version you encountered the problem
`develop` at ad68aaae3

### Friendica source (git, zip)
GIT

### PHP version
_Not relevant._

### SQL version
_Not relevant._

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #8498 
Part of #4090 

- Add new route contact/<cid>/poke and update all references
- Rework template with form field includes
- [frio] Enable modal behavior

Oof, this one was a doozy. I removed a bunch of stuff that was either not working (name autocomplete) or not used anymore (parent parameter). On Frio, I copied most of the modal behavior of private messages.

### CODE REVIEW

File: src/Module/Contact/Poke.php [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+		}
+
+		$activity = Activity::POKE . '#' . urlencode($verbs[$verb][0]);
+
+		$contact_id = intval($parameters['id']);
+		if (!$contact_id) {
+			return self::postReturn(false);
+		}
+
+		Logger::info('poke: verb ' . $verb . ' contact ' . $contact_id);
```
Comment (annando) [inline src/Module/Contact/Poke.php:45]: Could you remove the "poke:" from the logging string? We did this long before to see the current module.

File: src/Module/Contact/Poke.php [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		$arr['private']       = $private;
+		$arr['object-type']   = Activity\ObjectType::PERSON;
+
+		$arr['origin']        = 1;
+		$arr['body']          = '[url=' . $actor['url'] . ']' . $actor['name'] . '[/url]' . ' ' . $verbs[$verb][2] . ' ' . '[url=' . $contact['url'] . ']' . $contact['name'] . '[/url]';
+
+		$arr['object'] = '<object><type>' . Activity\ObjectType::PERSON . '</type><title>' . XML::escape($contact['name']) . '</title><id>' . XML::escape($contact['url']) . '</id>';
+		$arr['object'] .= '<link>' . XML::escape('<link rel="alternate" type="text/html" href="' . $contact['url'] . '" />') . "\n";
+
+		$arr['object'] .= XML::escape('<link rel="photo" type="image/jpeg" href="' . $contact['photo'] . '" />') . "\n";
```
Comment (annando) [inline src/Module/Contact/Poke.php:95]: I guess you haven't checked, if this is still needed? Before accepting the PR, I want to have a look at the receiver as well. When we don't parse that object at all, we don't need it. (And in the past we got rid of most of this stuff. I have the feeling as if the only really important part is the `verb` field.

File: src/Module/Contact/Poke.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Contact/Poke.php:45]: Absolutely.

File: src/Module/Contact/Poke.php (hunk 2 above)
Comment (MrPetovan) [reply src/Module/Contact/Poke.php:95]: No, I didn't touch this part at all. I was able to verify a poke item is being created and is private when it's supposed to.

Comment (annando) [reply src/Module/Contact/Poke.php:95]: I'm sure that you hadn't touched this. But I'm always keen in optimizing stuff - and I like to remove unused stuff. Something that isn't there can't make problems.

Comment (annando) [reply src/Module/Contact/Poke.php:95]: Update: It seems as if the object is indeed used. So we sadly can't remove it.

File: src/Module/Contact/Poke.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Contact/Poke.php]: Done.

**Pull request merged**